cdisc_rules_engine/services/data_services/local_data_service.py

```diff
@@ -94,6 +94,31 @@ def get_dataset_metadata(
         }
         return pandas.DataFrame.from_dict(metadata_to_return)
 
+    @cached_dataset(DatasetTypes.METADATA.value)
+    def get_dataset_contents_metadata(
```

Comment:
Why not use existing [get_variables_metadata](https://github.com/cdisc-org/cdisc-rules-engine/blob/2528e05eaa9ee2f734874d82312635189b2949c3/cdisc_rules_engine/services/data_services/local_data_service.py#L143) instead of creating this new method?